lib/storage/providers/MemoryOnlyProvider.ts

```diff
@@ -127,6 +127,10 @@ const provider: StorageProvider = {
         return Promise.resolve(_.keys(store));
     },
 
+    getAllEntries() {
+        return Promise.resolve(Object.entries(store));
```

Comment:
NAB - why `Object` here instead of `_` as above in `getAllKeys()`? Seems like the same effect either way?

Reply:
Generally we don't use underscore in TS files, and have lint rules to prevent it. but the lint config in Onyx is a bit different from E/App - most of our TS-specific lint rules are in E/App and should be moved to our ESLint plugin.